ui-tests/__test__/flows/feedback-form/render-upvote-result.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'playwright/test';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+import testIds from '../../../../src/helper/test-ids';
+
+export const renderUpvoteResult = async (page: Page, skipScreenshots?: boolean): Promise<void> => {
+  await page.evaluate((body) => {
+    const selectedTabId = window.mynahUI.getSelectedTabId();
+    if (selectedTabId != null) {
+      window.mynahUI.updateStore(selectedTabId, {
+        chatItems: [],
+      });
+
+      window.mynahUI.addChatItem(selectedTabId, {
+        type: 'answer' as any,
+        snapToTop: true,
+        body: 'This message is votable.',
+        canBeVoted: true,
+      });
+    }
+  });
+  await waitForAnimationEnd(page);
+
+  const thumbsUp = page.locator(getSelector(testIds.chatItem.vote.upvoteLabel));
+  expect(thumbsUp).toBeDefined();
+  await thumbsUp.click();
```

Comment:
nit: should we check (with expect(..)) also for what happens after the click? namely the UI change with thanks message?

Reply:
We check this with the screenshot already, so no point in double testing this really.